tests/config/nextflow.config

```diff
@@ -0,0 +1,35 @@
+params {
+    outdir = "output/"
+    publish_dir_mode = "copy"
+    enable_conda = false
+    singularity_pull_docker_container = false
+    max_cpus   = 2
+    max_memory = 6.GB
+    max_time   = 6.h
+}
+
+process {
```

Comment:
feels like some of this could be generalised and go to the template

Reply:
Yeah @maxulysse actually figured that out and added this to the `test.config` of Sarek so it's no longer necessary

```nextflow
// Enable container engines/virtualisation envs for CI testing
// only works when specified with the profile ENV
// otherwise tests can be done with the regular provided profiles
if (System.getenv('PROFILE')) {
    if ("$PROFILE" == "conda") {
        conda.createTimeout                      = "120 min"
        params.enable_conda                      = true
        charliecloud.enabled                     = false
        docker.enabled                           = false
        podman.enabled                           = false
        shifter.enabled                          = false
        singularity.enabled                      = false
    } else if ("$PROFILE" == "docker") {
        docker.enabled                           = true
        docker.userEmulation                     = { params.use_gatk_spark ? false : true }.call()
        charliecloud.enabled                     = false
        podman.enabled                           = false
        shifter.enabled                          = false
        singularity.enabled                      = false
    } else if ("$PROFILE" == "singularity") {
        params.singularity_pull_docker_container = false
        singularity.autoMounts                   = true
        singularity.enabled                      = true
        charliecloud.enabled                     = false
        docker.enabled                           = false
        podman.enabled                           = false
        shifter.enabled                          = false
    }
}
```

Reply:
Just going to wait for 2.1.0 https://github.com/nf-core/nascent/issues/84